setup.cfg

```diff
@@ -0,0 +1,152 @@
+[metadata]
+name = amsterdam-schema-tools
+version = 0.18.1
+url = https://github.com/amsterdam/schema-tools
+license = Mozilla Public 2.0
+author = Amsterdam Data en Informatie
+author_email = datapunt@amsterdam.nl
+description = Tools to work with Amsterdam schema.
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    jsonschema
+    schema
+    json
+    amsterdam
+    validation
+    code-generation
+classifiers =
+    Development Status :: 5 - Production/Stable
+    License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >= 3.7
+install_requires =
+    sqlalchemy ~= 1.3.23
```

Comment:
On other thought (and probably unrelated to this PR), looking at this massive requirements list makes me wonder if it would be nice to separate 'cli' requirements as well?
For example DSO api is not using CLI tools such as clock and pg-grant.

WDYT?

Reply:
Not sure. An optional requirements list is nice, but than the code should make provisions as well (catch import errors) and respond appropriately (whatever that may be) when optional dependencies are not present.

Taking it further in case  of `cli`. When that becomes optional, we should make the entry points (console apps) installation optional as well. Things get complicated rather quickly that way from a packaging point of view. In that case separating `cli` out of this project completely might be a better option.

`schematools` becomes `schematool-lib` (purely a library) and everything cli related would now become `schematools-cli` that has a dependency on `schematools-lib`. But that kind of a separation feels a little awkward.

So, I'm not sure. It definitely warrants more discussion.